### DISCUSSION TITLE
Country/state instead of signal reports in logbook view

### DISCUSSION THREAD

**Discussion by ofadam** [magicbug/Cloudlog#790] [Ideas / Feature Requests]
On various logbook views, signal report given/received get columns in the table, but country and/or state does not. I know there's limited horizontal space available, but would love to at least have country instead of RST.

It's entirely possible that this request is unique to me - but I wanted to put it out there in case others would find it useful.

**Comment by magicbug** [comment]
Theres now the option to change these.

**Reply by ofadam** [reply]
Saw that and love the new feature! Thanks!
